de.json

```diff
@@ -0,0 +1,260 @@
+    "Copy": "Kopieren",
+    "Paste Here": "Einfügen",
+    "Rename": "Umbenennen",
+    "Exclude from backups": "Von Sicherungen ausschließen",
+    "Download file here": "Datei herunterladen",
+    "Create Archive": "Archiv erstellen",
+    "Create Directory": "Ordner erstellen",
+    "Trash": "Papierkorb",
+    "Created": "Erstellt",
+    "Last Modified": "Zuletzt bearbeitet",
+    "Editable": "Bearbeitbar",
+    "Downloadable": "Herunterladbar",
+    "Full Path": "Vollständiger Pfad",
+    "Excluded from backups": "Von Sicherungen ausgeschlossen",
+    "Download": "Herunterladen",
+    "Calculate md5 hash": "MD5-Hash berechnen",
+    "Edit": "Bearbeiten",
+    "Move to trash": "Löschen",
+    "Are you sure you want to move the selected item to the trash?": "Möchten Sie das ausgewählte Element wirklich in den Papierkorb verschieben?",
+    "Delete Item": "Eigenschaft löschen",
+    "Rename Directory": "Ordner umbenennen",
+    "Please enter a new directory name": "Bitte geben Sie einen neuen Verzeichnisnamen ein",
+    "Rename File": "Datei umbenennen",
+    "Please enter a new filename": "Proszę wprowadzić nową nazwę pliku",
+    "New Directory": "Neuer Ordner",
+    "Please enter a name for the new directory": "Bitte geben Sie einen Namen für den neuen Ordner ein",
+    "No files to paste": "Keine Dateien zum Einfügen",
+    "You have no files in the clipboard to paste": "Sie haben keine Dateien in der Zwischenablage zum Einfügen",
+    "Want help fast?": "Brauchen Sie schnelle Hilfe?",
+    "Join CubeCoders on Discord!": "Tritt CubeCoders auf Discord bei!",
+    "Otherwise, for support with AMP please visit the": "Andernfalls besuchen Sie für Hilfe mit AMP bitte das ",
+    "support board": "Hilfe-Forum",
+    "Always make sure to include the information listed below.": "Achten Sie immer darauf, die unten aufgeführten Informationen anzugeben.",
+    "AMP is up-to-date.": "AMP ist aktuell.",
+    "Currently running version": "Derzeit laufende Version",
+    "View Latest Release Notes": "Neueste Versionshistorie",
+    "Diagnostics Data": "Diagnosedaten",
+    "Platform": "Plattform",
+    "System Type": "Systemtyp",
+    "Virtualization": "Virtualisierung",
+    "Loaded Plugins": "Geladene Plugins",
+    "Application Name": "Anwendungsname",
+    "Application Version": "Anwendungsversion",
+    "Restart this AMP instance": "Diese AMP-Instant neustarten",
+    "Upgrade this AMP instance": "Diese AMP-Instanz aktualisieren",
+    "Confirm Upgrade": "Aktualisierung bestätigen",
+    "Upgrading this instance may take up to 5 minutes. You will not be able to log into any instances that depend on this ADS instance.": "Die Aktualisierung dieser Instanz kann bis zu 5 Minuten dauern. Sie können sich nicht bei Instanzen anmelden, die von dieser ADS-Instanz abhängen.",
```

Comment:
Die Aktualisierung dieser Instanz kann bis zu 5 Minuten dauern. Du kannst dich in keine Instanz anmelden, die von dieser ADS Instanz abhängig ist.